### DISCUSSION TITLE
Blurred screen when steaming

### DISCUSSION THREAD

**Discussion by soul0nsky** [MonitorControl/MonitorControl#1564] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

After installing the app, problems with Discord began. In HD streaming mode, the Macbook Pro M1 screen and the Huawei MateView 34 monitor both have a very blurry picture. FPS displays 5 frames per second. Devices are connected via Thunderbolt. Do video codecs possibly conflict? How can you help or comment on the situation? 
Reinstalling the software didn’t help.

**Comment by waydabber** [comment]
Honestly I don't have a clear answer, not sure how MonitorControl would affect streaming adversely. Did you try https://github.com/waydabber/BetterDisplay - do you have the same issue with it?
